docs/conf.py

```diff
@@ -307,6 +307,9 @@ def _do_not_evaluate_in_jax(
 
 
 # -- Extension configuration -------------------------------------------------
+# Define prompt text pattern to be removed for copybutton
+copybutton_prompt_text = r">>> |\.\.\. |\$ |In \[\d*\]: | {2,5}\.\.\.: | {5,8}: "
```

Comment:
It would be worth adding a comment linking to https://sphinx-copybutton.readthedocs.io/en/latest/use.html#using-regexp-prompt-identifiers, assuming that was the source

Reply:
Thanks, done!